dev/ci/ci-vscoq.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download vscoq
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "$CI_BUILD_DIR/vscoq/language-server"
+  dune build && dune runtest
```

Comment:
Please use separate lines

Also I think you need to use -p to get dune to work when called in a subdirectory of coq